app/src/main/java/ch/epfl/sdp/MainActivity.java

```diff
@@ -72,6 +72,7 @@ public void logout() {
         MainActivity.killGame();
         authenticationController.signOut();
         new DatabaseHelper(this).deleteAllUsers();
+        LoginFormActivity.loggedUser = null;
```

Comment:
Why this?

Reply:
Because if we did not reset loggedUser to null when we logged out, next time when we open the app, we will log in directly.

Reply:
Oh, it's the logout, my bad